src/ui/components/HeroSection/index.js

```diff
@@ -32,6 +33,7 @@ export default class HeroSection extends React.Component<Props> {
         {linkTo ? (
           <Link
             className="HeroSection-link-wrapper"
+            onClick={(e) => this.props.onClick(e, linkTo)}
```

Comment:
We could also pass `onClick` directly here, and `bind` the `onExperimentClick` method in the parent component. That would be better because otherwise, how do we know that `linkTo` will be passed to the parent component here?

Reply:
hmm.I thought I tried something like that but it wasn't working. Let me recheck this tho..

Reply:
I'd say we have to make `onExperimentClick` a non-bound method:

```js
onExperimentClick() {
}
```

and call `bind` in the `onClick` prop:

```
onClick={this.onExperimentClick.bind(this, url)}
```

Reply:
for some reason bind wasn't working for me. I have it working in a slightly different way. let me know if that way is ok..

Reply:
looks good